site-src/geps/gep-713.md

```diff
@@ -1,25 +1,35 @@
-# GEP-713: Policy Attachment
+# GEP-713: Metaresources and Policy Attachment
 
 * Issue: [#713](https://github.com/kubernetes-sigs/gateway-api/issues/713)
 * Status: Experimental
 
 ## TLDR
 
-This GEP aims to standardize policy attachment to resources associated with
-Gateway API by establishing a pattern which defines how `Policy` API types can
-have their relevant effects applied to network traffic. Individual policy APIs
-(e.g. `TimeoutPolicy`, `RetryPolicy`, etc) will include a common `TargetRef`
-field in their specification to identify how and where to apply that policy.
+This GEP aims to standardize attachment of one Kubernetes object to another in
+Gateway API by establishing a pattern which defines how metaresources such as
+`Policy` API types can have their relevant effects applied to network traffic.
+Metaresources are a type of object that _augments_ the behavior of an object
+in a standard way.
+
+"Policy Attachment" is a specific type of metaresource that controls how policy
+can flow _across_ multiple objects in a hierarchy.
```

Comment:
I think the definition of "metaresource" above is pretty solid, but I'm thinking there might be some way to be more descriptive here... I know it's the TLDR, but maybe there's some way we can sneak in a brief example of what we mean? Just a thought.